File: tests/openmp_examples/devices/async_target.4.c
#pragma omp parallel num_threads(2)
#pragma omp single
#pragma omp task depend(out:v1)
#pragma omp task depend(out:v2)
#pragma omp target nowait depend(in:v1,v2) depend(out:p) map(to:v1,v2) map( from: p)
#pragma omp parallel for private(i)
#pragma omp task depend(in:p)

File: tests/openmp_examples/devices/async_target.4.f90
!$omp    parallel num_threads(2)
!$omp       single
!$omp          task depend(out:v1)
!$omp          end task
!$omp          task depend(out:v2)
!$omp          end task
!$omp          target nowait depend(in:v1,v2) depend(out:p) map(to:v1,v2) map(from: p)
!$omp          parallel do
!$omp          end target
!$omp          task depend(in:p)
!$omp          end task
!$omp      end single
!$omp    end parallel
